services/osparc-gateway-server/Dockerfile

```diff
@@ -88,7 +89,19 @@ COPY --chown=scu:scu services/osparc-gateway-server/requirements/_base.txt .
 # install base 3rd party dependencies (NOTE: this speeds up devel mode)
 RUN --mount=type=cache,mode=0777,target=/root/.cache/pip \
   pip install \
-  -r _base.txt
+  -r _base.txt \
+  && pip list -v
+
+# in ARM64 mode there is a catch: the piwheels package does not contain the dask-gateway-proxy executable in 64-bit
+RUN dpkgArch="$(dpkg --print-architecture)";\
+  case "$dpkgArch" in \
+  arm64) git clone --depth 1 --branch 0.9.0 https://github.com/dask/dask-gateway.git \
```

Comment:
Should the 0.9.0 be kept in sync with the client version?

Reply:
that might be a thing yes